application/frontend/controllers/ResetController.php

```diff
@@ -0,0 +1,93 @@
+<?php
+namespace frontend\controllers;
+
+use common\models\Reset;
+use common\models\User;
+use frontend\components\BaseRestController;
+use yii\filters\AccessControl;
+use yii\helpers\ArrayHelper;
+use yii\web\BadRequestHttpException;
+use yii\web\NotFoundHttpException;
+
+class ResetController extends BaseRestController
+{
+    /**
+     * Access Control Filter
+     * NEEDS TO BE UPDATED FOR EVERY ACTION
```

Comment:
Come to think of it, should these "NEEDS TO..." comments have a `@todo` stuck on them?